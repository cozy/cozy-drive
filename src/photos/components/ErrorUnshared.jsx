import React from 'react'
import ErrorShare from 'components/Error/ErrorShare'
import { Main } from 'cozy-ui/react/Layout'
import styles from '../targets/public/index.styl'
import classNames from 'classnames'

const ErrorUnsharedComponent = () => (
  <div
    data-test-id="pho-public-layout"
    className={classNames(
      styles['pho-public-layout'],
      styles['pho-public-layout--full'],
      'u-pt-3'
    )}
  >
    <Main className="u-pt-1-half">
      <ErrorShare errorType={`public_album_unshared`} />
    </Main>
  </div>
)

export default ErrorUnsharedComponent
