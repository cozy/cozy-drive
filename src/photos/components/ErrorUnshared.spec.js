import React from 'react'
import { shallow } from 'enzyme'
import ErrorUnshared from './ErrorUnshared'

describe('ErrorUnshared', () => {
  test('should match the snapshot', () => {
    const errorUnsharedInstance = shallow(<ErrorUnshared />)
    expect(errorUnsharedInstance).toMatchSnapshot()
  })
})
