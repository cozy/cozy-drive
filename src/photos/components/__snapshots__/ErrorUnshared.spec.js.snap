// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorUnshared should match the snapshot 1`] = `
<div
  className="u-pt-3"
  data-test-id="pho-public-layout"
>
  <Main
    className="u-pt-1-half"
  >
    <Wrapper
      errorType="public_album_unshared"
    />
  </Main>
</div>
`;
